src/utilities/qstring.c

```diff
@@ -389,6 +395,9 @@ char *qstrdup_between(const char *str, const char *start, const char *end) {
     int len = e - s;
 
     char *buf = (char *) malloc(sizeof(char) * (len + 1));
+	if (buf == NULL)
+		return NULL;
+
     strncpy(buf, s, len);
```

Comment:
maybe we can update this to memcpy() as well

Reply:
SGTM. How about using [qmemdup](https://github.com/wolkykim/qlibc/blob/6db37023044c0a4c4fbd50448eaef6b58d250af6/src/utilities/qstring.c#L406)?

Reply:
Your right. I'll change to memcpy.